src/configuration.ts

```diff
@@ -602,6 +616,9 @@ export function loadAtomistConfig(): AutomationServerOptions {
  * ATOMIST_TEAM environment variable, which takes precedence over the
  * configuration "teamdIds", which takes precedence over cfg.teamIds,
  * which may be undefined, null, or an empty array.
+ *
+ * @deprecated The usage of teamIds is deprecated and will be removed
```

Comment:
Should this refer to `resolveTeamIds` instead of just `teamIds`?

Reply:
Fixing